### PR TITLE
Issue #3 expose threadpools

### PR DESCRIPTION
**Pull request by dkichler**
Actually addresses both Issue #2 and Issue #3

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -2,3 +2,5 @@
 addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.4.8")
 
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.0")
+
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.1")
```
Comment (OldSneerJaw) [inline project/plugins.sbt]: What is this used for?

**Review by OldSneerJaw — COMMENTED** [review]
Seems straightforward for the most part, but there's an unexplained sbt dependency.

**Comment by dkichler** [comment]
I bungled this branch a tad - the dependency you commented on actually belongs to #10.  I've cleaned this PR/branch up.

**Review by OldSneerJaw — APPROVED** [review]
👩‍🚀

**Pull request merged**